pkg/new-ui/v1beta1/authzn.go

```diff
@@ -0,0 +1,116 @@
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/kubeflow/katib/pkg/util/v1beta1/env"
+	"github.com/pkg/errors"
```

Comment:
Should we stop using this library since this has been archived?

https://github.com/pkg/errors

Reply:
Sure, I'll switch to Go’s built-in errors package

Reply:
@apo-ger 
Probably, you forget to change to the standard errors library in this section.